genai-perf/tests/test_optuna_objective_generator.py

```diff
@@ -1,227 +0,0 @@
-# Copyright 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
why was this deleted?

Reply:
It would only be used by optimize and it relies heavily on config options that do not exist. If/When we decide to implement optimize we can pull these files back in to the code base.